subprojects/docs/src/docs/userguide/scalaPlugin.adoc

```diff
@@ -290,7 +290,7 @@ Incremental compilation requires dependency analysis of the source code. The res
 Note that Zinc's Nailgun based daemon mode is not supported. Instead, we plan to enhance Gradle's own compiler daemon to stay alive across Gradle invocations, reusing the same Scala compiler. This is expected to yield another significant speedup for Scala compilation.
 
 [[sec:scala_cross_compilation]]
-=== Compiling and testing for Java 6
+=== Compiling and testing for Java 6/7
```

Comment:
:o: (nitpick) "Java 6 or Java 7" would be preferred as a bit clearer

Reply:
Done.